be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -349,7 +348,7 @@ Status SegmentIterator::_init_return_column_iterators() {
             ColumnIteratorOptions iter_opts;
             iter_opts.stats = _opts.stats;
             iter_opts.use_page_cache = _opts.use_page_cache;
-            iter_opts.file_reader = _file_reader.get();
```

Comment:
I think the previous code maybe want to improve the performance by using raw pointer.